__tests__/app.test.js

```diff
@@ -263,4 +263,27 @@ describe("DELETE /api/comments/:comment_id", ()=>{
   test("400: should respond with a 400 status on invalid comment_Id", () => {
     return request(app).delete("/api/comments/fakeId").expect(400);
   });
-})
\ No newline at end of file
+})
+
+describe("GET /api/users", () => {
+  test("Responds with an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const {
+          body: { users },
+        } = response;
+        users.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
+        });
+      });
+  });
+  test("404: Should respond with 404 when provided an non-existent path", () => {
+    return request(app)
+      .get("/api/userss")
+      .expect(404)
```

Comment:
This error is not specific to this endpoint, if you have already tested a spelling mistake on the path in a previous test then this should be removed as it's the same error logic as before. A non-existent path is referred to as a _general_ error